examples/build_examples.sh

```
@@ -0,0 +1,81 @@
+#!/bin/sh
```

I'd add
```
set -o errexit
set -o pipefail
set -o nounset
```
to fail hard and prevent usage of undeclared variables